client/service/src/builder.rs

```diff
@@ -487,8 +491,36 @@ where
 	)
 	.map_err(|e| Error::Application(Box::new(e)))?;
 
+	let sysinfo = sc_sysinfo::gather_sysinfo();
+
+	info!("💻 Operating system: {}", TARGET_OS);
+	info!("💻 CPU architecture: {}", TARGET_ARCH);
+	if !TARGET_ENV.is_empty() {
+		info!("💻 Target environment: {}", TARGET_ENV);
+	}
```

Comment:
Could we not also just do all this inside the sysbench crate?

Reply:
Good idea. Done.